src/Adapter/Kpi/PendingDiscussionThreadsKpi.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter\Kpi;
+
+use ConfigurationKPI;
+use HelperKpi;
+use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use PrestaShop\PrestaShop\Core\Kpi\KpiInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class PendingDiscussionThreadsKpi implements KpiInterface
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var LegacyContext
+     */
+    private $context;
+
+    public function __construct(TranslatorInterface $translator, LegacyContext $context)
+    {
+        $this->translator = $translator;
+        $this->context = $context;
+    }
+
+    public function render(): string
+    {
+        $time = time();
+
+        $helper = new HelperKpi();
+        $helper->id = 'box-pending-messages';
+        $helper->icon = 'mail';
+        $helper->color = 'color1';
+        $helper->href = $this->context->getAdminLink('AdminCustomerThreads');
+        $helper->title = $this->translator->trans('Pending Discussion Threads', [], 'Admin.Catalog.Feature');
+        if (ConfigurationKPI::get('PENDING_MESSAGES') !== false) {
```

Comment:
Im pretty sure ConfigurationKPI::get() typehint is wrong (it should be string|false), as it returns false when configuration is not found. I wonder why php stan is not complaining, because if it would only return string this statement would always be falsy. Maybe you could try to fix the `get` method typehint (I think it shouldn't impact too much)?